script/feature_compiler_helper.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
I think a more appropriate place for this file is `brave/chromium_src/python_modules`, you can look at the structure and how other .py files are used there.

Reply:
Done. Thanks for the advice!